uportal-war/src/main/java/org/jasig/portal/rest/SessionRESTController.java

```diff
@@ -58,6 +60,11 @@ public void setPersonManager(IPersonManager personManager) {
         this.personManager = personManager;
     }
     
+    @Autowired(required = true)
```

Comment:
I haven't looked -- is an `IdentitySwapperManager` always available in all uPortal implementations, or is the way I disable identity swapping to fail to instantiate an `IdentitySwapperManager`?